docs/user/Dependencies.md

```diff
@@ -13,7 +13,7 @@ SOURCE | NAME | VERSION
 ~ | docker | any
 ~ | git | any
 ~ | kustomize | 3.7.0
-~ | kubectl | 1.19.9
+~ | kubectl | 1.20+
```

Comment:
I am kinda ok with this. Right now we support kubernetes 1.20 -1.22 Might want to make this more reflective of that value as folks could run with 1.23 and 1.24 at this time which is not supported; however, the 1.20+ leads one to believe this code is on the leading edge of kubernetes support.

Reply:
Updated the kubectl version to a range 1.20 -1.22.